CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+### pg_auto_failover v1.6.1 (July 7, 2021) ###
+
+This release contains monitor schema changes, so the monitor extension gets
+a version bump from 1.5 to 1.6, and this is the first release in the 1.6
+series.
+
+In this release we introduce a new state in the FSM: "dropped". This state
+allows a node to realise it's been dropped from the monitor, and act
+accordingly (mostly, stops cleanly and register it's been dropped).
+
+#### Added
+* Feature crash recovery before pg rewind (#656)
+* Allow building pg_auto_failover with Postgres 14. (#716)
+* Fix Postgres 14 beta2 incompatibility. (#739)
```

Comment:
This bug was never present in a released version, so we can skip this from the release notes.

Reply:
I wanted to show we've been following along for Postgres 14 ;-) that said, removed.